### DISCUSSION TITLE
TuringCourses/short_term/ics/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#58] [General]
# TuringCourses/short_term/ics/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/short_term/ics/

**Comment by ZhouTimeMachine** [comment]
学到姜老师的部分的时候建议可以不听，一方面你们基本已经会了，另一方面你可能需要花很多精力在寻找姜老师讲课的错误中（笑）

**Reply by NeilAlfred** [reply]
2023 年探子来报，现在姜 sir 讲课的那几天会详细地阐述数据通路，从而说明 CPU 的原理，甚至讲到总线、和一丁点的流水线原理，比前几年体验好多了。并且由于是中文讲课，许多听不懂 patt 讲课的同学终于有机会不用坐牢；

**Comment by dieKarotte** [comment]
真的超级推荐计概这门课程，虽然对于大一的你稍有难度（反正现在23级以后都在一春夏修完了计逻，但你以后修习计组、体系、操作系统的时候真的会感到那颗子弹终于正中眉心，在大Q老师的班上认真听完了每一节助教课程，收获满满，认真搞懂数据通路、中断、CPU组成，对后面的课程太有帮助了，觉得对于后面三大件真的很有概括性帮助，这大概正是概论课的魅力吧。给大Q老师磕一个orz
